stable/prometheus/templates/NOTES.txt

```diff
@@ -45,18 +47,19 @@ http://{{ . }}
 
 {{- else }}
 Get the server URL by running these commands in the same shell:
-{{- if contains "NodePort" .Values.alertmanager.serviceType }}
+{{- if contains "NodePort" .Values.alertmanager.service.type }}
   export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ template "alertmanager.fullname" . }})
   export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
   echo http://$NODE_IP:$NODE_PORT
 
-{{- else if contains "LoadBalancer" .Values.alertmanager.serviceType }}
+{{- else if contains "LoadBalancer" .Values.alertmanager.service.type }}
   NOTE: It may take a few minutes for the LoadBalancer IP to be available.
         You can watch the status of by running 'kubectl get svc --namespace {{ .Release.Namespace }} -w {{ template "alertmanager.fullname" . }}'
+
   export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ template "alertmanager.fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
-  echo http://$SERVICE_IP:{{ .Values.alertmanager.httpPort }}
-{{- else if contains "ClusterIP"  .Values.alertmanager.serviceType }}
-  export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ template "fullname" . }},component={{ .Values.alertmanager.name }}" -o jsonpath="{.items[0].metadata.name}")
+  echo http://$SERVICE_IP:{{ .Values.alertmanager.service.servicePort }}
```

Comment:
output on the notes is difficult to read.  Maybe some slight changes in newlines and prefixing with AlertManager / Prometheus Server on the lines to segment

```
NOTES:
The Prometheus server can be accessed via port 80 on the following DNS name from within your cluster:
prometheus-prometheus-server.infra.svc.cluster.local


Get the server URL by running these commands in the same shell:
  export POD_NAME=$(kubectl get pods --namespace infra -l "app=prometheus,component=server" -o jsonpath="{.items[0].metadata.name}")
  kubectl --namespace infra port-forward $POD_NAME 9090
The Prometheus alertmanager can be accessed via port 80 on the following DNS name from within your cluster:
prometheus-prometheus-alertmanager.infra.svc.cluster.local


Get the server URL by running these commands in the same shell:
  export POD_NAME=$(kubectl get pods --namespace infra -l "app=prometheus,component=alertmanager" -o jsonpath="{.items[0].metadata.name}")
  kubectl --namespace infra port-forward $POD_NAME 9093

For more information on running Prometheus, visit:
https://prometheus.io/
```